python-package/setup.py

```diff
@@ -344,7 +344,7 @@ def run(self):
           extras_require={
               'dask': [
                   'dask[array]>=2.0.0',
-                  'dask[dataframe]>=2.0.0'
+                  'dask[dataframe]>=2.0.0',
```

Comment:
oh wow, thank you!

Reply:
TBH, first time I noticed that was at LGTM site:
https://lgtm.com/projects/g/microsoft/LightGBM?mode=tree